src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier randomColor = new ColorSupplier();
+
+        Figure circle = new Circle(randomColor.getRandomColor(), "", new Random().nextInt(20) + 1);
+        Figure isoscelesTrapezoid = new IsoscelesTrapezoid(randomColor.getRandomColor(),
+                "", new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                new Random().nextInt(20) + 1);
+        Figure rectangle = new Rectangle(randomColor.getRandomColor(),"",
+                new Random().nextInt(20) + 1, new Random().nextInt(20) + 1);
+        Figure rightTriangle = new RightTriangle(randomColor.getRandomColor(), "",
+                new Random().nextInt(20) + 1, new Random().nextInt(20) + 1);
+        Figure square = new Square(randomColor.getRandomColor(), "", new Random().nextInt(20) + 1);
```

Comment:
The constructors for the figures are being called with an empty string as the second parameter. Ensure that this matches the expected parameters for each figure's constructor. If the second parameter is not supposed to be an empty string, you should provide the appropriate value.